Documentation/admission-controller-usage.md

```diff
@@ -15,11 +15,8 @@ Enabling the Rook admission controller is recommended to provide an additional l
 To deploy the Rook admission controllers we have a helper script that will automate the configuration.
 
 This script will help us achieve the following tasks
-1. Creates self-signed certificate.
-1. Creates a Certificate Signing Request(CSR) for the certificate and gets it approved from the Kubernetes cluster.
-1. Stores these certificates as a Kubernetes Secret.
-1. Creates a Service Account, ClusterRole and ClusterRoleBindings for running the webhook server with minimal privileges.
-1. Creates ValidatingWebhookConfig and fills the CA bundle with the appropriate value from the cluster.
+1. Creates certificate using cert-manager.
+2. Creates ValidatingWebhookConfig and fills the CA bundle with the appropriate value from the cluster.
 
 Run the following commands:
 ```console
```

Comment:
I don't think `config-admission-controller.sh` is needed anymore. We can just document that k8s 1.16 or newer is required, no need for the additional check just for that. Let's update these instructions to call `deploy_admission_controller.sh`.

Reply:
In the next PR to make this as simple as possible, I wonder if we can even get rid of the scripts and just document kubectl commands and have yaml file(s) in the examples folder.

Reply:
> In the next PR to make this as simple as possible, I wonder if we can even get rid of the scripts and just document kubectl commands and have yaml file(s) in the examples folder.

Yes, I believe we'll get rid of all the scripts in the next PR(:crossed_fingers: )

Reply:
> I don't think `config-admission-controller.sh` is needed anymore. We can just document that k8s 1.16 or newer is required, no need for the additional check just for that. Let's update these instructions to call `deploy_admission_controller.sh`.

removed file `config-admission-controller.sh` and updated the doc to point `tests/scripts/deploy_admission_controller.sh`.